### PR TITLE
docs: added illustration section

### PR DESCRIPTION
**Pull request by sadekbazaraa**
**New**

My first attempt at adding net new pages. Illustration... 
This is a WIP, please don't merge.

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -48,3 +48,60 @@
 //---------------------------------------
 @import '../components/ColorBlock/color-block';
 @import '../components/ColorGrid/color-grid';
+
+//---------------------------------------
+// Table component
+//---------------------------------------
+@import "narrative";
```
Comment (vpicone) [inline src/styles/index.scss]: We might want to do something different here, do you remember what/who added this file?

Comment (sadekbazaraa) [reply src/styles/index.scss]: @vpicone this was in particular regard to the way this table is being done on the narrative page on the Events site.  https://www.ibm.com/design/event/narrative

I was going to go back and at least change the name to be more reflective. My guess is that this was a bit of a hack from the 29 team. I wasn't sure how to implement it but saw that the Events say was calling some CSS for the table to work so I replicated what they did and wanted to ask you about it.

Mike also pointed out that in mobile that we need a line break between header and copy. Is there a better way we can achieve the same result? This is a pattern that I'm sure we'll want to duplicate across sites.

IMG_4408 (1)

**Comment by sadekbazaraa** [comment]
@vpicone we're ready to make the Illustration pages live. 

FYI @mjabbink

**Comment by vpicone** [comment]
@sadekbazaraa should be live in 10-15 min

**Pull request merged**